pynest/examples/gap_junctions_two_neurons.py

```diff
@@ -22,27 +22,23 @@
 """
 Gap Junctions: Two neurons example
 ------------------
-This is a simple example of two hh_psc_alpha_gap neurons connected
-by a gap-junction. Please note that gap junctions are two-way connections:
-In order to create an accurate gap-junction connection between two
-neurons i and j the use of the flag "make_symmetric" is required.
+This script simulates two Hodgkin-Huxley neurons of type `hh_psc_alpha_gap`
+connected by a gap junction. Both neurons receive a constant current of
+`100.0 pA`. The neurons are initialized with different membrane potentials and
+synchronize over time due to the gap-junction connection.
 """
 
 import nest
-import pylab
+import pylab as pl
 import numpy
 
 nest.ResetKernel()
 
-nest.SetKernelStatus({'resolution': 0.05,
-                      # Settings for waveform relaxation
-                      # 'use_wfr': False uses communication in every step
-                      # instead of an iterative solution
-                      'use_wfr': True,
-                      'wfr_comm_interval': 1.0,
-                      'wfr_tol': 0.0001,
-                      'wfr_max_iterations': 15,
-                      'wfr_interpolation_order': 3})
```

Comment:
@heplesser The wfr settings are now discussed in the general gap junction documentation ( http://nest-simulator.org/simulations-with-gap-junctions/ ).  The default settings are well chosen and from my point of view there is only little reason to change these settings. Until now the parameter were (re-)set to their defaults in the examples just to demonstrate how the parameters are set. For anyone just looking at the examples it might however of course be useful to have this code lines here.

After further considerations I reintroducted the settings in one of the examples (the network example).